features/steps/backup-restore.py

```diff
@@ -90,10 +118,9 @@ def step_impl(context):
     assert_that(response, not_(has_key("FunctionError")))
 
 
-def run_fargate_with_envfile(envfile, image="paddlehq/mysql-backup-s3"):
+def run_fargate_with_envfile(envfile, image="paddlehq/mysql-backup-s3", call_env=[]):
```

Comment:
Not sure you should define `call_env=[]` here - better to have `call_env=None` and set to a list if `is None`

Reply:
@chrisbarr thank mate  stupid mistake ; in my defence I started with a clever tuple thingy but then changed it.  I added flake8-bugbear to stop myself from repeating this.